Source/SwiftLintBuiltInRules/Rules/Performance/EmptyCountRule.swift

```diff
@@ -59,6 +97,67 @@ private extension EmptyCountRule {
             }
         }
     }
+
+    final class Rewriter: ViolationsSyntaxRewriter {
+        private let operators: Set = ["==", "!=", ">", ">=", "<", "<="]
+        private let configuration: EmptyCountConfiguration
+
+        init(configuration: EmptyCountConfiguration,
+             locationConverter: SourceLocationConverter,
+             disabledRegions: [SourceRange]) {
+            self.configuration = configuration
+            super.init(locationConverter: locationConverter, disabledRegions: disabledRegions)
+        }
+
+        override func visit(_ node: SequenceExprSyntax) -> ExprSyntax {
```

Comment:
This override shouldn't be needed with #5428 being merged.

Reply:
Fixed at a501697 (& `git rebase main` ed)